src/mame/netronics/exp85.cpp

```diff
@@ -201,6 +243,43 @@ void exp85_state::sod_w(int state)
 	}
 }
 
+/* Memory mapping mgmt during reset and interrupt */
+
+void exp85_state::status_out(u8 status)
+{
+	// In the real hardware this is monitored via the IO/M, S0, S1 
+	// and ALE output pins of the 8085.
+	// Since these pins are not emulated, then we must explicitly get the internal 
+	// interrupt acknowledge state (0x23 or 0x26) and behave as the monitor expects.
+	auto current_pc = m_maincpu->pc();
+	if (status == 0x23 || status == 0x26)
+	{
+		// When an interrupt is triggered the low memory shall be set to the ROM mirror
+		LOG("set low memory to ROM mirror (PC 0x%x, proc status 0x%x)\n", current_pc, status);
```

Comment:
Use LOGMASKED here so that CPU cycles aren't being blown logging this in normal operation.

Reply:
Removed logging, does not make much sense to leave it there at this point.